components/toolbar/style.scss

```diff
@@ -35,6 +35,10 @@ ul.components-toolbar {
 			height: $icon-button-size - 16px;
 		}
 	}
+
+	.block-editable__inline-toolbar & {
```

Comment:
I was unsure about this, but leaned towards the toolbar because we've been wanting to keep the Editable minimal. I'll move it.